exercises/practice/pythagorean-triplet/pythagorean-triplet.spec.js

```diff
@@ -58,7 +58,7 @@ describe('Triplet', () => {
     ]);
   });
 
-  test.skip('triplets for large number', () => {
+  xtest('triplets for large number', () => {
```

Comment:
If it still takes ~ 16 s, we can't "unskip" this, because the web UI has a maximum of 10 s of run time. Apart from that, all these changes are great!

Reply:
Would it be possible to run the test on a smaller input?

Reply:
an input of 25000 takes ~9.5s and returns [ [ 5000, 9375, 10625 ] ]. Or do you mean i should mark the test as skipped but keep the code changes?

Reply:
That last thing! So people can run it likely but it won't run in the web.